### PR TITLE
cars app created and first api on cars app

### PR DESCRIPTION
**Pull request by ViorelStavaru**

### CODE REVIEW

File: myproject/myproject/urls.py
Comment (ionescucosti) [inline myproject/myproject/urls.py:1]: remove the code below =======
try to run the app before you commit

File: myproject/cars/urls.py
Comment (ionescucosti) [inline myproject/cars/urls.py:1]: you should use "home" instead of "index"

File: myproject/myproject/urls.py
Comment (ViorelStavaru) [reply myproject/myproject/urls.py:1]: removed

File: myproject/cars/urls.py
Comment (ViorelStavaru) [reply myproject/cars/urls.py:1]: replaced, it will be visible with the next commit

**Comment by ViorelStavaru** [comment]
I'll push the changes with the next commit

**Pull request closed without merging**